helpers.go

```diff
@@ -17,19 +17,20 @@
 package btrfs
 
 /*
-#include <stddef.h>
-#include <btrfs/ioctl.h>
-#include <btrfs/ctree.h>
+#include "btrfs.h"
 */
 import "C"
 
 import (
 	"bufio"
 	"bytes"
+	"encoding/binary"
 	"fmt"
 	"os"
 	"strings"
 	"unsafe"
+
+	"github.com/yalue/native_endian"
```

Comment:
Slightly wondering if this is a bit of a "leftpad" dependency we're introducing here (looks like it's.. 3 lines of code effectively?) Wondering if we should just implement something similar in this repository

Reply:
^^ @AkihiroSuda wdyt? Dependency looks very trivial, and a bit on the fence if we really want to introduce a new dependency for this.

Reply:
Updated the PR to use `x/sys` instead.
`x/sys` v0.5.0 was released with https://github.com/golang/sys/commit/13fe000ef0b8edcf0ceca490fb5574c344013bc8 just yesterday.

Reply:
oh nice! I was about to suggest "perhaps this is something golang/x/sys would accept